mongodb/src/test/java/org/hibernate/ogm/datastore/mongodb/test/dialectinvocations/GridDialectOperationInvocationsTest.java

```diff
@@ -121,8 +121,6 @@ public void insertGetOfEntityAndElementCollectionAndDelete() throws Exception {
 				"getAssociation",
 				"getTuple",
 				"getAssociation",
-				"removeAssociation",
```

Comment:
Yes, they are included in the `executeBatch()` call. I agree on the need for a testing facility for that.

Created a new issue here to track this: https://hibernate.atlassian.net/browse/OGM-1071